python/phylanx/util/__init__.py

```diff
@@ -35,6 +35,27 @@ def phy_print(m):
     else:
         print("ndim=", ndim)
 
+def get_node(node,**kwargs):
+    args = [arg for arg in ast.iter_child_nodes(node)]
+    params = {"name":None,"num":None}
+    for k in kwargs:
+        if k not in params:
+            raise Exception("Invalid argument '%s'" % k)
+        else:
+            params[k] = kwargs[k]
+    name = params["name"]
+    num  = params["num"]
+    if name != None:
+        for i in range(len(args)):
+            a = args[i]
+            nm = a.__class__.__name__
+            if nm == name:
+                if num == None or num == i:
+                    return a
+    elif num != None:
+        if num < len(args):
+            return args[num]
+
```

Comment:
Missing `else`?

Reply:
The intent was to return None in the else case. That's what happens the way the code is written.

Reply:
Ok, understood. Wouldn't it be better to be explicit about this, however?